### PR TITLE
Remove `individual_offer_conditions` feature flag

### PR DESCRIPTION
**Pull request by despo**
… and code that is no longer relevant

## Context
Remove the `individual_offer_conditions` feature flag, and all of the code that is no longer relevant

## Link to Trello card

https://trello.com/c/heOvktbx/4001-remove-individual-conditions-feature-flag-and-old-implementation
## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Review by steventux — APPROVED** [review]
:+1:

File: app/views/provider_interface/conditions/confirm_update.html.erb [hunk 1]
```diff
@@ -1,8 +0,0 @@
-<% content_for :browser_title, title_with_error_prefix('Confirm conditions', @form_object.errors.any?) %>
-<% content_for :before_content, govuk_back_link_to(provider_interface_application_choice_path(@application_choice.id)) %>
-
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <%= render ProviderInterface::ConditionStatusBulkReviewPageComponent.new(form_object: @form_object, application_choice: @application_choice) %>
```
Comment (benswannack) [inline app/views/provider_interface/conditions/confirm_update.html.erb:6]: I think we can fully remove the all the components with "Bulk" in the name too

File: app/controllers/provider_interface/conditions_controller.rb [hunk 2]
```diff
@@ -1,66 +0,0 @@
-module ProviderInterface
-  class ConditionsController < ProviderInterfaceController
```
Comment (benswannack) [inline app/controllers/provider_interface/conditions_controller.rb:2]: Should we remove/redirect the old routes?

**Review by benswannack — APPROVED** [review]
Given this a test and it's looking good :)

One question inline

File: app/controllers/provider_interface/conditions_controller.rb (hunk 2 above)
Comment (despo) [reply app/controllers/provider_interface/conditions_controller.rb:2]: I removed the routes and added a temporary redirect for conditions/edit that we can remove in a month or so.

File: config/routes.rb [hunk 3]
```diff
@@ -662,9 +662,8 @@
       get '/timeline' => 'application_choices#timeline', as: :application_choice_timeline
       get '/emails' => 'application_choices#emails', as: :application_choice_emails
       get '/feedback' => 'application_choices#feedback', as: :application_choice_feedback
-      get '/conditions' => 'conditions#edit', as: :application_choice_edit_conditions
-      patch '/conditions/confirm' => 'conditions#confirm_update', as: :application_choice_confirm_update_conditions
-      patch '/conditions' => 'conditions#update', as: :application_choice_update_conditions
+      # TODO: Remove redirect after 1 Sept 2021
+      get '/conditions' , to: redirect('/provider/applications/%{application_choice_id}/offer/conditions')
```
Comment (benswannack) [inline config/routes.rb:666]: Should this redirect to `/id/condition-statuses/edit`?

Comment (despo) [reply config/routes.rb]: Done. I grepped the routes for `conditions` and missed that path.

**Pull request merged**